cloudify_awssdk/common/decorators.py

```diff
@@ -135,6 +135,16 @@ def wrapper_inner(**kwargs):
                     ctx.instance.runtime_properties[
                         'resource_config'] = resource_config
                     ctx.instance.runtime_properties[EXT_RES_ID] = resource_id
+                    _, _, _, operation_name = ctx.operation.name.split('.')
+                    if operation_name not in ['delete', 'create'] and \
```

Comment:
@EarthmanT Why we need just to ignore `delete` and `create`

Reply:
For all resources, even existing resources, we do not expect this property to exist during `create` or during `delete`. But already in `configure`, `start`, and `stop`, we do expect it to exist.
I'm not crazy about this method - but I'm not sure of another way to make sure we do not run this function in those operations.